analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.LearnConstraint.cs

```diff
@@ -442,5 +442,51 @@ public void Branching_LearnsObjectConstraint_ConstantPattern_Other(string expres
             validator.ValidateTag("Else", x => x.Should().BeNull());
             validator.ValidateTag("End", x => x.Should().BeNull());
         }
+
+        [DataTestMethod]
+        [DataRow("arg is not not object")]
```

Comment:
What about the "IsType" operation? Do you track it or is it implemented already?

Reply:
#6041